lib/trino-filesystem-azure/src/main/java/io/trino/filesystem/azure/AzureOutputFile.java

```diff
@@ -69,9 +69,7 @@ public OutputStream create(AggregatedMemoryContext memoryContext)
     public void createOrOverwrite(byte[] data)
             throws IOException
     {
-        try (OutputStream out = createOutputStream(newSimpleAggregatedMemoryContext(), true)) {
-            out.write(data);
-        }
+        blobClient.upload(BinaryData.fromBytes(data), true);
```

Comment:
There's a version of this method that takes a timeout. We do configure write timeouts in the HTTP client, and I don't want to add another config option, so I didn't use it here.